packages/@lwc/integration-karma/test/native-shadow/Event-methods/Event.composedPath.spec.js

```diff
@@ -1,6 +1,23 @@
 import { createElement } from 'lwc';
 import Synthetic from 'x/synthetic';
 
+/**
+ * Returns a promise that resolves with the composed path of an event dispatched from `fromTarget` to `toTarget`.
+ *
+ * @param {EventTarget} toTarget - The target element where the event listener is added.
+ * @param {EventTarget} fromTarget - The target element from which the event is dispatched.
+ * @returns {Promise<EventTarget[]>} A promise that resolves with the composed path of the event.
+ */
+function getComposedPath(toTarget, fromTarget) {
+    return new Promise((resolve) => {
+        toTarget.addEventListener('test', (event) => {
+            resolve(event.composedPath());
+        });
+
+        fromTarget.dispatchEvent(new CustomEvent('test', { bubbles: true, composed: true }));
+    });
+}
```

Comment:
I couldn't resist adding this helper function as the exact same code was used in every test here.